test/utils/url.test.js

```diff
@@ -0,0 +1,72 @@
+import { appendQuery } from 'src/utils';
+import nock from 'nock';
+import expect from 'expect';
+
+/**
+ * @private
+ * Adapted from https://github.com/lakenen/node-append-query
+ */
+describe('appendQuery', function(){
+    it('should append querystring to queryless url', function () {    
+        var result = appendQuery('http://example.com/foo', 'bar=baz&beep=boop')
+            , expected = 'http://example.com/foo?bar=baz&beep=boop'
```

Comment:
Much of the test code is written using es6. We don't use arrow functions for the callback in the `it()` function because this breaks the scope that is provided from whom called the callback function. 

If in a `beforeEach()` I want to set a variable like the following
```javascript
beforeEach(function() {
   this.foo = 'bar';
});
```
And then I would like to access that variable in one of my `it()` functions I must use `function() {}` and not an arrow function.

This is one example why we use regular functions.